### ISSUE TITLE
platformio terminal not working

### ISSUE THREAD

**Issue by Debojyoti1915001** [platformio/platformio-atom-ide-terminal#851]
- [ ] I have reset Atom to defaults prior to submitting report.
- [ ] I have not reset Atom to defaults prior to submitting report.
### Steps to reproduce

1.
2.
3.

**Expected behavior:** 

**Actual behavior:** 

**Reproduces how often:** 

### Versions

```
> atom --version

```
```
> apm --version

```
OS name and version:
Platformio-ide-terminal version:

**Comment by jovancosta-zz** [comment]
same here, with no error code.
the message:
copyright (c) Microsoft Corporation. All rights reserved
is not loaded to me.
==
I tried installing it manually, previous versions, and also inside the own atom software and it didn't work.
I'm using windows 10.
Doing the installation manually I got 2 errors. but unfortunately, I've lost these logs. Now it's not appearing any error to me.
Were you able to solve your issue?
